src/NorthwindWebsite/appsettings.json

```diff
@@ -39,5 +39,12 @@
   "EmailSenderConfigs": {
     "SendersEmail": "evg.tsarkov@gmail.com",
     "SendersName": "Northwind Training App"
+  },
+  "AzureAd": {
+    "Instance": "https://login.microsoftonline.com/",
+    "ClientId": "50bf8629-aec9-4712-8576-3e2bf3e3625f",
```

Comment:
I wanna hope these creds are fake. In general, if you see that some sensitive data should be stored in app settings, in commit messages you should **always** provide fake values